src/main/java/org/folio/consortia/repository/UserTenantRepository.java

```diff
@@ -33,4 +34,11 @@ public interface UserTenantRepository extends JpaRepository<UserTenantEntity, UU
   @Modifying
   @Query("DELETE FROM UserTenantEntity ut WHERE ut.userId= ?1 AND ut.isPrimary= true")
   void deleteByUserIdAndIsPrimaryTrue(UUID userId);
+
+  @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.userId NOT IN (SELECT ut.userId FROM UserTenantEntity ut WHERE ut.isPrimary=true)")
+  Optional<List<UserTenantEntity>> getByUserIdAndIsPrimaryFalse();
```

Comment:
We can do this logic without optional, userId should be added as parameter

Reply:
fixed